client.go

```diff
@@ -76,6 +76,9 @@ func (w *worker) stop() {
 
 // Client is a wrapper around *amqp.Connection which allow interaction with one queue/exchange.
 type Client struct {
+	// LogMessagesMaxSize indicate max size in bytes of messages, which are logged.
+	LogMessagesMaxSize int
```

Comment:
Where is this variable initialized? You mean that the client should know it? I'm not sure if this is a good idea

Reply:
nowhere, if client want to enable logging then
```golang
client.LogMessagesMaxSize = 1000
```